dali/test/operator_argument.h

```diff
@@ -27,6 +29,48 @@ struct InputArg {
   // TODO(mszolucha): For Dali's argument input
 };
 
+template <typename T, typename Enable = void>
+struct TestOpArgToStringImpl {
```

Comment:
Usually in cases like this, I'd reverse the relationship between `to_string` and printing.
Then we'd have a uniform implementation of `to_string` (below) and wouldn't need to create intermediate strings when printing.
```
template <typename T>
std::string to_string(std::ostream &s, TestOpArgValue<T> &t) {
  std::stringstream ss;
  ss << t;
  return ss.str();
}
```